.github/workflows/compile-kfp.yml

```diff
@@ -0,0 +1,79 @@
+name: Compile KFP pipelines
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - "kubeflow-pipelines/docling-standard/**"
+      - "kubeflow-pipelines/docling-vlm/**"
+      - ".github/workflows/compile-kfp.yml"
+  push:
+    branches: [ main ]
+    paths:
+      - "kubeflow-pipelines/docling-standard/**"
+      - "kubeflow-pipelines/docling-vlm/**"
+      - ".github/workflows/compile-kfp.yml"
+
+permissions:
+  contents: read
+  actions: write
+
+jobs:
+  compile:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: docling-standard
+            dir: kubeflow-pipelines/docling-standard
+            out: docling_convert_pipeline_compiled.yaml
```

Comment:
Looks like you're going to have to change names once this PR merges  https://github.com/opendatahub-io/odh-data-processing/pull/12/files#